src/components/Reports/GoalCalculator/SummaryReport/MpdGoal/MpdGoalTable.tsx

```diff
@@ -42,7 +42,13 @@ export const MpdGoalTable: React.FC<MpdGoalTableProps> = ({ goal }) => {
   const { t } = useTranslation();
   const locale = useLocale();
   const localeText = useDataGridLocaleText();
-  const calculations = useGoalLineItems(goal);
+  const calculations = useGoalLineItems({
+    netMonthlySalary: goal.netMonthlySalary,
+    taxesPercentage: goal.taxesPercentage,
+    rothContributionPercentage: goal.rothContributionPercentage,
+    traditionalContributionPercentage: goal.traditionalContributionPercentage,
+    ministryExpensesTotal: goal.ministryExpensesTotal,
+  });
```

Comment:
I think this can be simplified too.

```suggestion
  const calculations = useGoalLineItems(goal);
```